Tests/CatalogItems/Write-RsFolderContent.Tests.ps1

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+Describe "Write-RsFolderContent" {
+
+    Context "Write-RsFolderContent with min parameters"{
+        $folderName = 'SutWriteRsFolderContentMinParameters' + [guid]::NewGuid()
+        New-RsFolder -Path / -FolderName $folderName
+        $folderPath = '/' + $folderName
+        $localReportPath =   (Get-Item -Path ".\").FullName  + '\Tests\CatalogItems\testResources'
+        Write-RsFolderContent -Path $localReportPath -RsFolder $folderPath
+        
+        It "Should upload a local report in Report Server" {
+            $uploadedReport = (Get-RsFolderContent -RsFolder $folderPath -Recurse ) | Where-Object TypeName -eq 'Report'
+            $uploadedReport.TypeName | Should Be 'Report'
```

Comment:
>$uploadedReport.TypeName | Should Be 'Report' [](start = 12, length = 45)

I recommend you to check for the name of the report instead of type name as $uploadedReport is supposed to be an item of type 'Report'.